pyspecdata/log_test.py

```diff
@@ -0,0 +1,11 @@
+from pyspecdata import *
+import logging
+#logging.basicConfig(stream=sys.stdout,level=logging.DEBUG)
+#logger.info("hello")i
+
+
+
+mylogger = init_logging("info")#print("hello")
+
+logger.info("Something, something, something, dark side...")
```

Comment:
when I run, I get an error in my result
````
INFO:pyspecdata.core:Something, something, something, dark side...
--- Logging error ---
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\logging\__init__.py", line 1028, in emit
    stream.write(msg + self.terminator)
AttributeError: 'str' object has no attribute 'write'
Call stack:
  File "log_test.py", line 4, in <module>
    logger.info("Something, something, something, dark side...")
Message: 'Something, something, something, dark side...'
Arguments: ()
--> log_test.py(4):pyspecdata.core             <module> 2020-05-27 20:06:57,138
INFO: Something, something, something, dark side...
```

Reply:
to me, this says that when it expects a `stream` object, it's getting a string for some reason instead

Reply:
So with the new changes that I just pushed, I get this error:
```
Traceback (most recent call last):
  File "log_test.py", line 2, in <module>
    mylogger = init_logging("info")
  File "c:\users\alex guinness\gitstuff\pyspecdata\pyspecdata\general_functions.py", line 134, in init_logging
    if type(level) is str:
UnboundLocalError: local variable 'level' referenced before assignment
```

Reply:
My last commit actually worked.  I'm confused by what exactly you did.  It looks like you copied some but not all my code without actually pulling in the changes -- I will shoot you an image of what I see in gitk on slack